WinUIGallery/DataModel/ControlInfoData.json

```diff
@@ -21,7 +21,7 @@
           "Content": "",
           "IsNew": false,
           "IsUpdated": false,
-          "HideSourceCodeLinks": true,
```

Comment:
https://github.com/microsoft/WinUI-Gallery/pull/1184 already addresses this by hiding the Related Controls and Docs buttons when their arrays are empty - so then we wouldn't need this bool?

Reply:
Well we are also hiding the source code links with this option since they don't make much sense for these sections. I don't think there is an option to do this without a special flag on the control page.

Reply:
Yes true, but I wonder if we could do the same.

Thinking about it.. are there particular pages where hiding the source code links? Seems to be pretty useful always right?

Reply:
I don't think there are special pages beside the design guidance where it would make a lot of sense. Thinking about it, the a11y section might actually also benefit from that. The reason for those links is that users often wandered where the source code is for the samples and the sample code below the samples often cannot capture the entire context.